src/compiler/diagnosticMessages.json

```diff
@@ -1701,7 +1701,14 @@
         "category": "Error",
         "code": 2653
     },
-
+    "Proper external module that carries external typings cannot contain tripleslash references.": {
+        "category": "Error",
+        "code": 2654
+    },
+    "Proper external module that carries external typings should be '.d.ts' file.": {

```

Comment:
External package typings have to be `.d.ts` files. Please contact the package author to update the package definition.